greenery/models/staging/stg_addresses.sql

```diff
@@ -0,0 +1,24 @@
+
+{# I would not enable an update on addresses as #}
+{# this would generate data inconsistency on older instances. #}
```

Comment:
Wouldn't it be the opposite ? If there is an address change, we want to keep historical value for the past orders and new value for the future orders

Reply:
I wasn't sure that an address change would result in an update or in a new `address_id`. For the former it makes sense to create a snapshot. For the latter, probably not. I think I was expecting an `updated_at` timestamp for the latter...

Reply:
updated_at - fair assumption. I guess to clarify this point we would need more about the source tables.